File: app/src/main/java/com/example/android/airqualitymonitor/properties/Pm2_5.java

package com.example.android.airqualitymonitor.properties;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Pm2_5 {

    @SerializedName("v")
    @Expose
    private Double v;

    public Double getV() {
        return v;
    }

    public void setV(Double v) {
        this.v = v;
    }

}

File: app/src/main/java/com/piyushsatija/pollutionmonitor/model/properties/Pm2_5.kt
package com.piyushsatija.pollutionmonitor.model.properties

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Pm2_5 {
    @SerializedName("v")
    @Expose
    var v: Double? = null

}